install_vendored_dynamic_framework/after/Pods/Target Support Files/RealmSwift/RealmSwift.xcconfig

```diff
@@ -1,8 +1,8 @@
 CONFIGURATION_BUILD_DIR = ${PODS_CONFIGURATION_BUILD_DIR}/RealmSwift
 FRAMEWORK_SEARCH_PATHS = $(inherited) "${PODS_ROOT}/Realm/realm-swift-0.94.1/ios/swift-1.2"
 GCC_PREPROCESSOR_DEFINITIONS = $(inherited) COCOAPODS=1
-OTHER_LDFLAGS = -framework "Realm"
-OTHER_SWIFT_FLAGS = $(inherited) "-D" "COCOAPODS"
+OTHER_LDFLAGS = $(inherited) -l"c++" -framework "Realm"
```

Comment:
I am uncertain if these should be part of an xcconfig of a pod that does not include them. I know they are harmless as they are system libs and frameworks but I think it would be cleaner if we do not include them.